modular_skyrat/modules/encounters/code/nri_raiders.dm

```diff
@@ -498,3 +500,138 @@ GLOBAL_VAR(first_officer)
+	add_objective(new /datum/objective/fortify)
+	add_objective(new /datum/objective/survive)
+	for(var/datum/mind/M in members)
+		var/datum/antagonist/cop/R = M.has_antag_datum(/datum/antagonist/cop)
+		if(R)
+			R.objectives |= objectives
+
+/datum/objective/policing
+	name = "policing"
+	explanation_text = "Contact the station to perform an inspection. Delegate responsibilities among the ship's crew. Minimise civilian casualties."
+	martyr_compatible = TRUE
+
+/datum/objective/inspect_area
+	name = "inspect area"
+	explanation_text = "Inspect certain department and make sure it's up to our specifications. Special scrutiny and pickyness is advised."
+	var/inspection_area
+	martyr_compatible = TRUE
+
+/datum/objective/inspect_area/New(text)
+	. = ..()
+	inspection_area = pick("Cargo","Engineering","Security","Command","Service","Medical","Science")
+
+/datum/objective/inspect_area/update_explanation_text()
+	..()
+	if(inspection_area)
+		explanation_text = "Inspect [inspection_area] department and make sure it's up to our specifications. Special scrutiny and pickyness is advised."
+	else
+		explanation_text = "Perform a general station inspection and make sure it's up to any loose specifications you can think of."
+
+/datum/objective/survey
+	name = "survey"
+	martyr_compatible = TRUE
+	admin_grantable = TRUE
+	var/survey_area
+
+/datum/objective/survey/New(text)
+	. = ..()
+	survey_area = pick("Cargo","Engineering","Security","Command","Service","Medical","Science")
+
+/datum/objective/survey/update_explanation_text()
+	..()
+	if(survey_area)
+		explanation_text = "Execute continuous crime prevention and citizen surveying procedures over [survey_area] department. Prevent crime in a given department and perform a public survey to collect people's opinions on various matters."
+	else
+		explanation_text = "Execute continuous crime prevention and citizen surveying procedures around the station. Prevent crime around the area and perform a public survey to collect people's opinions on various matters."
+
+/datum/objective/steal_n_of_type/contraband
+	name = "confiscate contraband"
+	explanation_text = "Confiscate at least cool number pieces of contraband. Drugs, illicit weaponry, armor or equipment of any sort."
+	amount = 5
+
+/datum/objective/steal_n_of_type/contraband/New()
+	. = ..()
+	amount = rand(10,25)
+	explanation_text = "Confiscate at least [amount] pieces of contraband. Drugs, illicit weaponry, armor or equipment of any sort."
+	update_explanation_text()
+	return
+
+/datum/objective/steal_n_of_type/contraband/check_completion()
+	return completed ///I am letting them roleplay this out.
+
+/datum/objective/fortify
+	name = "fortify"
+	explanation_text = "Establish an outpost orbiting the station."
+	martyr_compatible = TRUE
+
+/datum/objective/fortify/New()
+	. = ..()
+	explanation_text = "Establish an outpost orbiting the [station_name()]."
+	update_explanation_text()
+	return
```

Comment:
I'm really not a big fan of this one, honestly. It feels like this could be its own thing rather than being bundled into this.

Reply:
> I'm really not a big fan of this one, honestly. It feels like this could be its own thing rather than being bundled into this.

Meant to give them some long-term goal so they don't die of boredom; otherwise they'll probably get drunk in the bar out of lack of direction. Which isn't inherently bad, of course, but I doubt people'll develop their role any further than the other objectives.